docs/daisy-migrating-existing-workflows.md

```diff
@@ -0,0 +1,110 @@
+# Migrating Image Creation Workflows to Daisy
+
+This page describes how to use Daisy to automate creation of custom GCE images.
+These images are usually based on an existing base image (often a fresh install
+of your OS of choice), with some extra software you have chosen to install. This
+page assumes you have an existing workflow for creating such an image, and
+describes how to use Daisy to automate this task.
+
+To begin using Daisy for your existing workflows, follow these steps:
+
+  1. Choose a base image to start with.
+  2. Enumerate all the steps you currently follow to prepare your image.
+  3. Make sure all those steps can be done from the command line.
+  4. Write a new (or edit an existing) script to run on your base image which
+     does *all* of the steps you identified in step 2.
+  5. Write a Daisy workflow which creates at least one disk (the first disk
+     should be your base image), starts an instance from that disk, runs your
+     script, waits for the script to finish, then takes an image from the disk.
+
+The rest of this page describes these steps in greater depth.
+
+## Selecting a Base Image
+
+Your base image can come from anywhere, but if you're starting with an OS that
+is available on Compute Engine, then you can use what Daisy calls a "partial
```

Comment:
Partial URL is a term from the API, a Daisy workflow is essentially just a workflow of API calls (the json is mostly just passed through after some minor setup/validation). A partial URL is just the url without the common https://www.googleapis.com/compute/v1/

See sourceImage here:
https://cloud.google.com/compute/docs/reference/latest/disks

Reply:
Cool, added a link to https://cloud.google.com/compute/docs/reference/latest/disks#sourceImage to the doc and made it sound less like partial URLs are unique to Daisy.